docs/providers/aws/guide/serverless.yml.md

```diff
@@ -0,0 +1,116 @@
+<!--
+title: Serverless Framework - AWS Lambda Guide - Serverless.yml Reference
+menuText: Serverless.yml
+menuOrder: 15
+description: A list of all available properties on serverless.yml for AWS
+layout: Doc
+-->
+
+<!-- DOCS-SITE-LINK:START automatically generated  -->
+### [Read this on the main serverless docs site](https://www.serverless.com/framework/docs/providers/aws/guide/serverless.yml)
+<!-- DOCS-SITE-LINK:END -->
+
+# Serverless.yml Reference
+
+Here is a list of all available properties in `serverless.yml` when the provider is set to `aws`.
+
+```yml
+# serverless.yml
+
+service: myService
+
+frameworkVersion: ">=1.0.0 <2.0.0"
+
+provider:
+  name: aws
+  runtime: nodejs4.3
+  stage: dev # Set the default stage used. Default is dev
+  region: us-east-1 # Overwrite the default region used. Default is us-east-1
+  profile: production # The default profile to use with this service
+  memorySize: 512 # Overwrite the default memory size. Default is 1024
+  timeout: 10 # The default is 6
+  deploymentBucket: com.serverless.${self:provider.region}.deploys # Overwrite the default deployment bucket
```

Comment:
IMHO it's worth to rephrase to "Deployment bucket name. Default is generated by the framework".